src/schematools/ckan/_convert.py

```diff
@@ -0,0 +1,76 @@
+from collections import defaultdict
+from time import strftime
+from typing import Any, Optional
+
+from ..types import DatasetSchema
+
+
+def from_dataset(ds: DatasetSchema) -> dict[str, Any]:
```

Comment:
I'd use a different name. e.g. `dataset_to_ckan()`? That "from_..*" notation works well as classmethod but now that name exists outside such scope

Reply:
The intention is to `import ckan` instead of `from ckan import from_dataset`.

Reply:
ok, perhaps ask a few others whether they agree. This will be a public API after all.

Reply:
We hebben net gestemd en het is drie tegen twee vóór deze optie, met de opmerking dat je altijd nog

    from schematools.ckan import from_dataset as dataset_to_ckan

kunt doen ;)